packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -50,6 +50,7 @@ import { CallControlOptions } from '../CallComposite/types/CallControlOptions';
 import { useUnreadMessagesTracker } from './ChatButton/useUnreadMessagesTracker';
 /* @conditional-compile-remove(gallery-layouts) */
 import { VideoGalleryLayout } from '@internal/react-components';
+import { _InternalCustomButtonType } from '../common/ControlBar/CustomButton';
```

Comment:
Looks like this is just imported but not used with onRenderButton below (missed save before committing?)

Reply:
Ah, I found that added it to onRenderButton causes this issue
![image](https://github.com/Azure/communication-ui-library/assets/97130533/b938647d-2aaf-4c9e-b3e2-7d04023ef887).